schemas/iso19139/src/main/plugin/iso19139/index-fields/default.xsl

```diff
@@ -91,6 +91,7 @@
   <!-- ========================================================================================= -->
   <xsl:template match="/">
     <Document locale="{$isoLangId}">
+      <xsl:apply-templates mode="xlinks"/>
```

Comment:
if you don't match all subchildren at some point, it will not traverse the tree. eg. match=".//@xlink:href" or by applying the same mode in the child mode.

An easier way is to add 
```
  <xsl:template mode="index"
                match="@xlink:href">
    <Field name="xlink" string="{.}" store="true" index="true"/>
  </xsl:template>
``` 
as the index mode is taking care of visiting the tree (see l124).

Maybe "xlink" for field name?